docs/get-started/python.md

```diff
@@ -0,0 +1,82 @@
+---
+parent: Get started
+---
+
+# Build SDKs for Python
+
+## Required tools
+
+- [Python 3.10.4](https://www.python.org/)
```

Comment:
Does it have to be these specific versions, or could you say Python 3.x?, pip 20.x?

Reply:
Updated in https://github.com/microsoft/kiota/pull/1792/commits/3b79f65263a4910ff6329073bfa23b4b1dd5ff7a